packages/react/src/callout/docs/overview.mdx

```diff
@@ -96,3 +96,19 @@ The feature variant of the callout can be used to draw even more emphasis to the
 	<Text as="p">Description of the callout.</Text>
 </Callout>
 ```
+
+## Title
+
+By default, `Callout` renders `CalloutTitle` as an `h2`. If you need to change the title’s semantic tag to an `h3`, for instance, you can use the `CalloutTitle` component in the `title` prop to explicitly set it. For information on maintaining proper heading levels refer to [Structuring headings and labels](/content/content-structure#structuring-headings-and-labels).
+
+```jsx
+import { CalloutTitle } from '@ag.ds-next/react/callout';
+
+<Callout
+	tone="info"
+	variant="feature"
+	title={<CalloutTitle as="h3">Submission successful</CalloutTitle>}
+>
+	<Text as="p">Your application has been successfully submitted.</Text>
```

Comment:
I'm confused. Isn't this the sort of statement we reserve for alerts, i.e. `PageAlert` and `SectionAlert`?

Reply:
lol Copy and paste!

Reply:
Fixed!